app/routers/vip_shoplist.py

```
@@ -9,9 +9,9 @@
 from __future__ import annotations
 
 from decimal import Decimal
-from typing import Annotated, Any, cast
+from typing import Annotated, Any, Optional, cast
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)